.github/workflows/release.yaml

```diff
@@ -28,6 +28,15 @@ jobs:
           args: release --rm-dist
         env:
           GITHUB_TOKEN: ${{ secrets.ACCESS_TOKEN }}
+      - uses: azure/docker-login@v1
+        with:
+          login-server: index.docker.io
+          username: ${{ secrets.DOCKER_USER }}
+          password: ${{ secrets.DOCKER_PASSWORD }}
```

Comment:
Are docker credentials required? As we discussed last time, can we build the image locally and use it for CI testing?

Reply:
here we will define the DOCKER_USER in our secret that is nirmata